lntest/btcd.go

```diff
@@ -1,5 +1,3 @@
-// +build btcd
```

Comment:
we might need to include build tags to exclude this if bitcoind or neutrino is active